requirements.txt

```diff
@@ -54,7 +55,17 @@ notify2==0.3.1; sys_platform != 'win32' and sys_platform != 'darwin'
 # Uncomment line below or manually install after installing requirements.
 # pygobject>=3.10.2; sys_platform != 'win32' and sys_platform != 'darwin'
 
+# PyYAML does not have binary packages for Mac Users
+PyYAML==6.0.1 --no-binary=PyYAML; sys_platform == 'darwin'
+PyYAML==6.0.1; sys_platform != 'darwin'
+
+# Apprise Requirements
+requests==2.31.0
+requests-oauthlib==1.3.1
```

Comment:
I think we are missing some `requests` sub-dependencies, like `urllib3` and `idna`.

Reply:
I'm not sure if you are, i was able to send myself a (secure) ntfy and pushbullet notification which wrap the `requests` object.  I don't' spot anything abnormal off hand yet anyway.

Reply:
It's not required, as pip will fetch those depencies. But I need all the sub depencies in here too to have consistent builds. We had way too many times sub depencies updates that aren't noticed resulting breaking releases.

Reply:
done